### ISSUE TITLE
[confusing] Feedback from `/serverless/observability/apm-troubleshooting`

### ISSUE THREAD

**Issue by chrisdistasio** [elastic/observability-docs#3284]
## What do you see?

> Double check that the intake URL, API key, and **_SSL configuration_** are correct in your APM agent configuration

@simitt @axw is SSL config a valid for Agent -> MIS comms?

## What were you expecting to see?

**Comment by axw** [comment]
@chrisdistasio good catch, I don't think it's relevant for serverless. System TLS certs should work OOTB.
